packages/main/src/Card.ts

```diff
@@ -85,6 +87,24 @@ class Card extends UI5Element {
 	@slot({ type: HTMLElement, invalidateOnChildChange: true })
 	header!: Array<CardHeader>;
 
+	/**
+	 * Defines if a busy indicator would be displayed over the card.
```

Comment:
List and Table use "loading indicator" instead of "busy indicator". I think it will be better to use it here too

Reply:
Done